client/src/components/charts/pie/component.tsx

```diff
@@ -84,7 +85,7 @@ export const PieChart = <T extends unknown>({
                     }}
                   />
 
-                  {selected?.includes(arc.data.id) && (
+                  {(selected?.includes(arc.data.id) || arc.data.id === 1) && (
```

Comment:
Why `|| arc.data.id === 1`?

Reply:
Because we want to show the text when clicked (selected) or if not always show the % of the risked.

Reply:
But this is not a problem of the pie right?